Resources/Textures/Structures/Machines/VendingMachines/smartfridge.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/1516a728931b4985c1e86f0c5995a5aa1554a1ad and modified by Swept",
+  "copyright": "Taken from https://github.com/tgstation/tgstation/blob/master/icons/obj/vending.dmi modified by Swept",
```

Comment:
just leave the old link, copyright should link to a specific commit and never master